src/test.sh

```diff
@@ -458,11 +446,11 @@ run_test() {
 	commit "$author" "$email" "$subject" "$body" >/dev/null
 	[ "$merge" = 1 ] \
 		&& git switch "$HEAD_BRANCH" >/dev/null 2>&1 \
-		&& git merge --no-ff "$BASE_BRANCH" -m "Merge branch '$BASE_BRANCH' into '$HEAD_BRANCH'" >/dev/null 2>&1
+		&& git merge --no-ff "$BASE_BRANCH" -m "Merge branch '$BASE_BRANCH' into '$HEAD_BRANCH" >/dev/null 2>&1
 
 	set +e
 	local raw_output
-	raw_output=$("$CHECKER_SCRIPT" "$REPO_DIR" 2>&1)
+	raw_output=$(env bash "$CHECKER_SCRIPT" "$REPO_DIR" 2>&1)
```

Comment:
Is this not going to pick up the right shell based on shebang alone?

Reply:
Sorry for confusion. Yes, just changing shebangs is alright. :) Dropped it.